### ISSUE TITLE
Bug in linking within the ReadTheDocs docs

### ISSUE THREAD

**Issue by honzajavorek** [fralau/mkdocs_macros_plugin#71]
Hi, just came to learn about the plugin and noticed a bug. The following link is broken:

![Screenshot_2021-04-01 mkdocs-macros](https://user-images.githubusercontent.com/283441/113308523-8b3c3e00-9306-11eb-97b9-e24fc2ea28de.png)

I'm visualising the problem below by the diff syntax. The green is what should be in place, the red is what is there now:

```diff
+ https://mkdocs-macros-plugin.readthedocs.io/en/latest/pluglets/
- https://mkdocs-macros-plugin.readthedocs.io/en/pluglets
```

The markup in the source looks correct, so I don't know how to fix this. It's probably something in the settings of either MkDocs or ReadTheDocs.

**Comment by honzajavorek** [comment]
Meanwhile I found more of these, but I suppose it's the same problem. Perhaps the docs could be checked for broken links like this. I'm happy to contribute if we're able to identify the root cause together.

**Comment by fralau** [comment]
Fixed the "pluglet" link, as requested.

It's an idiosyncrasy of a mkdocs with index pages (the rules for relative links are different for index.md and the other pages).

**Comment by honzajavorek** [comment]
Great! I'm pretty sure I've spotted at least one more somewhere. I'll see your change and go through all the links during next week. If I find something, I'll send a PR with a fix, mimicking whatever you did to fix this one 🙂

**Comment by fralau** [comment]
Thanks a lot! You are welcome to propose a PR!

As a caution, there is something funny: on `mkdocs serve` the link may work, but the link may still be broken on the target static website (as produced by `mkdocs build`)

**Comment by fralau** [comment]
What do we do with this issue? Do we keep it open, or do we close it?

(I you propose a PR, you could still refer to it?)

**Comment by honzajavorek** [comment]
Hi, I have this as my TODO No. 1 item when I get to programming, but this week's difficult in that regard. Please, let's leave this open for a few more days, I definitely plan to get back to it when I get pass other tasks that distract me now.

**Comment by honzajavorek** [comment]
Found time today! I went through other links and discovered a few more broken ones. I created https://github.com/fralau/mkdocs_macros_plugin/pull/77 as a fix and added it alongside a [few other PRs](https://github.com/fralau/mkdocs_macros_plugin/pulls) I've sent week ago when reading the documentation. I think we can close this.

One thing though. I'm no MkDocs expert, but when learning about how to do internal links in MkDocs, [I've seen them](https://www.mkdocs.org/user-guide/writing-your-docs/#internal-links) with the `.md` extension. Is it possible that if you added `.md` to all your internal links, MkDocs would interpret the links correctly even for the index pages instead of us needing to think about the `../` discrepancy when writing the document? I didnt't try it, but I'd expect it to work this way.
